### ISSUE TITLE
Close log icon button lacks a label

### ISSUE THREAD

**Issue by henrycatalinismith** [zetkin/call.zetk.in#269]
## Steps to reproduce

1. Open an assignment.
2. Click the log button.
3. Press `⌘ + F5` to start VoiceOver.
4. Focus the close button.

## Expected outcome

The button should have a label describing its purpose as per [WCAG success criterion 1.1.1](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html).

## Actual outcome

There's no label so it's unclear what clicking it will do.

![VoiceOver with the button focused and the announcement just says button](https://user-images.githubusercontent.com/566159/191589508-505f4b5a-a7cc-470f-9299-1aec2634613c.png)

## Severity

The success criterion covering this issue is WCAG level A – the minimum level. Most organizations aim for a conformance level of AA, which includes all level A criteria. Any time an icon button is unlabeled it's generally a high level of severity as the affected functionality is effectively inoperable via a screen reader.

## Recommendation

Add an `aria-label` saying `Close` or something like that.
